doc/Development_Documentation/20_Extending_Pimcore/25_Implement_Your_Own_Search.md

```diff
@@ -0,0 +1,65 @@
+# Implement your own search
+
+## Register Implementation
+
+Pimcore provides two different registries (=facades) where you can register your custom implementation.
+The two registries are: `searchImplementationRegistry` and `quickSearchImplementationRegistry`.
```

Comment:
combine to one

Reply:
done c78c82a705653ad7f75262d03414a7ee9665b78c